src/System.Windows.Forms/src/System/Windows/Forms/ToolStrip.cs

```diff
@@ -1608,7 +1605,7 @@ public ToolStripRenderer Renderer
                     Renderer = ToolStripManager.CreateRenderer(RenderMode);
```

Comment:
We can replace this with.
```suggestion
                    _renderer = value;
                    _currentRendererType = _renderer.GetType();
                    OnRendererChanged(EventArgs.Empty);
```